### PR TITLE
fix(block, date-picker, list-item-group, panel, pick-list-group, popover, tip, tip-manager)!: Set default internal heading to a div.

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** #5099

## Summary

fix(block, date-picker, list-item-group, panel, pick-list-group, popover, tip, tip-manager)!: Set default internal heading to a div.

- Set default heading to be a div to accommodate more audiences
- Applies to block, date-picker, list-item-group, panel, pick-list-group, popover, tip, and tip-manager

### CODE REVIEW

File: t9nmanifest.txt [hunk 1]
```diff
@@ -29,4 +29,4 @@ src\components\shell-panel\assets\shell-panel\t9n
 src\components\time-picker\assets\time-picker\t9n
 src\components\tip-manager\assets\tip-manager\t9n
 src\components\tip\assets\tip\t9n
-src\components\value-list\assets\value-list\t9n
```
Comment (jcfranco) [inline t9nmanifest.txt:32]: Can you revert this unrelated change?

**Comment by jcfranco** [comment]
Devs will need to update their code to preserve heading levels, so this would be a breaking change, right?

**Comment by driskull** [comment]
> Devs will need to update their code to preserve heading levels, so this would be a breaking change, right?

Yes, I guess that would be a breaking change if they were not setting any value.

**Comment by driskull** [comment]
Updated to breaking change.

**Comment by driskull** [comment]
Created issue: https://github.com/Esri/calcite-components/issues/5768
